example/20-componentconfig-gardener-controller-manager.yaml

```diff
@@ -56,6 +56,9 @@ server:
     tls:
       serverCertPath: dev/tls/gardener-controller-manager.crt
       serverKeyPath: dev/tls/gardener-controller-manager.key
+shootBackup:
+  schedule: "0 * * * *"
+  maximum: 7
```

Comment:
Remove this since we no more use this. Earlier we had limit based garbage collection where we used to use this field for upper limit.  Now, by default its exponential snapshot which don't require `maximum` field anymore.  We didn't removed it earlier cause we were not sure about how to deal with API compatibility. But we are anyways changing API now, its best time to remove it.

Reply:
Okay, I will remove it.